wiki/Tournaments/OZT/2018/en.md

```diff
@@ -0,0 +1,114 @@
+# osu! Z-Tournament 2018
+![osu!Z-Tournament 2018](logo.png)
+
+- The **osu! Z-Tournament 2018**, is a 4v4 international osu!standard tournament.
+- There are no rank limits, therefore allowing any player to join.
+- Score system: **Scorev2**
+- This tournament will hold up to **64 teams, with 6 players at minimum and 8 players at maximum in each team.**
```

Comment:
This could go into the lead paragraph, be moved to the "stage instructions" (or "tournament rules" section), or both.